ARKitExample/ViewController.swift

```diff
@@ -0,0 +1,204 @@
+//
+//  ViewController.swift
+//  ARKitExample
+//
+//  Created by Lizzie Siegle on 8/10/17.
+//  Copyright © 2016-2017 Twilio, Inc. All rights reserved.
+//
+
+import UIKit
+import SceneKit
+import ARKit
+import TwilioVideo
+
+class ViewController: UIViewController {
+
+    @IBOutlet var sceneView: ARSCNView!
+
+    override var prefersStatusBarHidden: Bool {
+        return true
+    }
+
+    var accessToken = "TWILIO_ACCESS_TOKEN"
+    var room: TVIRoom?
+    weak var consumer: TVIVideoCaptureConsumer?
+    var frame: TVIVideoFrame?
+    var displayLink: CADisplayLink?
+
+    var supportedFormats = [TVIVideoFormat]()
+    var videoTrack: TVILocalVideoTrack?
+    var audioTrack: TVILocalAudioTrack?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Configure the ARSCNView which will be used to display the AR content.
+        sceneView.delegate = self
+
+        // Since we will also capture from the view we will limit ourselves to 30 fps.
+        sceneView.preferredFramesPerSecond = 30
+        // Since we are in a streaming environment, we will render at a relatively low resolution.
+        sceneView.contentScaleFactor = 1
+
+        // Show feature points, and statistics such as fps and timing information
+        sceneView.showsStatistics = true
+        sceneView.debugOptions =
+            [ARSCNDebugOptions.showFeaturePoints, ARSCNDebugOptions.showWorldOrigin]
+
+        // Create a new scene, and bind it to the view.
+        let scene = SCNScene(named: "art.scnassets/ship.scn")!
+        sceneView.scene = scene
+
+        // We only support the single capture format that our ARSession provides.
+        // Since ARSession doesn't seem to give us any way to query its output format, we will assume 1280x720x30
+        let format = TVIVideoFormat.init()
+        format.dimensions = CMVideoDimensions.init(width: 1280, height: 720)
```

Comment:
This is kind of sketchy. We definitely aren't getting 1280x720 by rendering at 1x resolution. Effectively our capturer only supports a single format, but I could probably make a better guess than this.